manifests/plugin/openscap/params.pp

```diff
@@ -6,4 +6,7 @@
   $listen_on               = 'https'
   $openscap_send_log_file  = '/var/log/foreman-proxy/openscap-send.log'
   $spooldir                = '/var/spool/foreman-proxy/openscap'
+  $contentdir              = '/usr/share/foreman-proxy/openscap/content'
```

Comment:
Not sure I understand your last comment.
When installing we create symlinks: `openscap/content`, `openscap/reports`, etc.
Shall I point to `openscap/content` or to the full path `/usr/share/foreman-proxy/openscap/content` ?